cmd/landscaper-controller/app/options.go

```diff
@@ -20,12 +20,19 @@ import (
 )
 
 type options struct {
-	log        logr.Logger
-	configPath string
-	deployers  string
-
-	config           *config.LandscaperConfiguration
-	enabledDeployers []string
+	log                logr.Logger
+	configPath         string
+	deployers          string
+	webhookNamespace   string
+	webhookName        string
+	disableWebhooks    string
+	webhookServerPort  int
+	webhookServicePort int32
```

Comment:
i think it would make sense to have seperate webhook options to keep the options more structured.
E.g.
```
type webhookOptions strut {
  namespace string
  ...
}

type options struct {
  log logr.Logger
  configPath string
  webhook webhookOptions
}
```

Reply:
then we should also validate all webhook values to fail early if something is missing